### ISSUE TITLE
'robots' meta tag detects 'noindex'.

### ISSUE THREAD

**Issue by newt0** [zzossig/hugo-theme-zzo#276]
I can't get my site indexed in Google Search Console.
This is because the 'robots' meta tag detects 'noindex'.
Where can I edit the robots meta tags?
I can only find the robots meta tag in the public directory.

**Comment by szpak** [comment]
There is a related solved issue - #234. The mentioned there [link to Hugo forum](https://discourse.gohugo.io/t/custom-robots-txt-and-sitemap-xml-templates/11869?u=szpak) was useful for me.

**Comment by zzossig** [comment]
Sorry, I don't know much about `robots.txt`.
All I know is that you can put your `robots.txt` file to your `root/static` folder.
And you can access it in the browser `https://yourdomain.com/robots.txt`
As @szpak said, it's better to get help in the Hugo forum

**Comment by newt0** [comment]
【Just sharing and reporting!】

After doing the following and waiting a few days, [my site](https://wasedabookshelf.com) started to show up in Google's search results.

1. Add a `robots.txt` file in the layouts directory
(`layouts/robots.txt`)
```layouts/robots.txt
User-agent: *
Allow: /

{{ range where .Data.Pages "Params.robotsdisallow" true }}
Allow: /
{{ end }}
```

2. Write `enableRobotsTXT = true' in `config.toml` in the `config` directory.
(`config//default/config.toml`)
```config//default/config.toml
baseURL = "http://wasedabookshelf.com/"
title = "早稲田の本棚"
theme = "zzo"

defaultContentLanguage = "ja"
defaultContentLanguageInSubdir = true
hasCJKLanguage = true

enableRobotsTXT = true // add

summaryLength = 70
buildFuture = true
・・・・・・(Abbreviation)
```

3. Write `enableRobotsTXT = true' in `config.toml` in the root directory.
(config.toml)

```config.toml
baseURL = "http://wasedabookshelf.com/"
languageCode = "ja-jp"
title = "早稲田の本棚"
theme = "zzo"
enableRobotsTXT = true // add
```

4. Add a `robots.txt` file in the layouts directory of the `themes` directory.
(`themes/zzo/layouts/robots.txt`)

```themes/zzo/layouts/robots.txt
User-agent: *
Allow: /

{{ range where .Data.Pages "Params.robotsdisallow" true }}
Allow: /
{{ end }}
```

5. Set up a custom domain by getting it from Google Domains instead of the URL given by netlify.
(Change the `baseURL` in `config.toml` as well)


Maybe we don't need to do 3 and 4.

Also, this method doesn't allow for Google's indexing by Google Search Console and Netlify URL, but anyway, it does show up in Google's search results.

In conclusion, I think the easiest way to get your site to show up in Google's search results is as follows

- Get a custom domain from Google Domains
- Setting up a custom domain obtained from Google Domains in Netlfiy
- Add a `robots.txt` file
- Write 'enableRobotsTXT = true' in `config.toml`
(※You'll have to wait a few days before Google's crawlers will recognize your site.)

**Comment by newt0** [comment]
@szpak @zzossig 
Thank you for your kind words!
